app/views/users/_form.html.erb

```diff
@@ -88,7 +88,8 @@
     <% end %>
 
     <div class='tab-pane' id='roles'>
-      <%= checkbox_f f, :admin if User.current.can_change_admin_flag? %>
+      <% caption = @user.inherited_admin? ? _('Admin rights are inherited from UserGroup') : '' %>
+      <%= checkbox_f f, :admin, help_text: caption if User.current.can_change_admin_flag? %>
```

Comment:
![Admin-message](https://user-images.githubusercontent.com/2884324/68574609-97520200-046a-11ea-89b0-e23c80bfbc38.png)
I belive `help_text` is a secondary label for checkbox and doesn't look so great in this situation. You can checkout the help options in `form_helper.rb`, generally there are `label_help` (toggable question mark on label), `help_inline` (help following the input - be aware this one is overriden by error message if the input is wrong) and `help_block` (this is a line under the input). I believe the best one here would be `help_block`, but if you like `help_inline` better, I wouldn't mind eighter.